concepticondata/concepticon.tsv

```diff
@@ -5,8 +5,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 4	COURTYARD	The house	An area wholly or partly surrounded by walls or buildings.	Person/Thing	
 5	GAZELLE	Animals	An antelope of the genus Gazella mostly native to Africa and capable of running at high speeds for long periods.	Person/Thing	
 6	EARTHQUAKE	The physical world	The violent shaking of the ground produced by deep seismic waves, originating from the epicentre.	Person/Thing	
-7	GATHER	Spatial relations	To heap up; to collect or gather (e.g. work, magazines, etc.).	Action/Process	
-8	CURSE	Religion and belief	An appeal or prayer for evil or misfortune to befall someone or something.	Action/Process	
+7	GATHER	Spatial relations	To collect or gather (e.g. work, magazines, etc.).	Action/Process	
+8	CURSE	Religion and belief	To wish for evil or misfortune to befall someone or something.	Action/Process	
```

Comment:
The previous definition for CURSE was indeed more related to the noun than to the verb, but "to curse" (which is the actual elicitation) is closer to "to cast" than "to wish" -- even in IE languages, you can trace back the second etymology of most comparanda to *preḱ- or *wenh1.

I would change it to something like "to imprecate evil or misfortune upon someone or something, usually by means of a prayer or implication".

Reply:
That's better.